charts/replicated-library/templates/classes/_secret.tpl

```diff
@@ -30,4 +30,5 @@ stringData:
 {{- with $values.data }}
   {{- tpl (toYaml .) $ | nindent 2 }}
 {{- end }}
+type: {{ $values.type}}
```

Comment:
```suggestion
type: {{ $values.type }}
```

Reply:
Nitpick formatting, sometimes spaces around {{ actually matter. Here I don't know that it does just keeping it clean.

Reply:
shoot, I forgot I could just click "commit suggestion"...